base/audit_events.go

```diff
@@ -72,6 +240,638 @@ var AuditEvents = events{
+	},
+	AuditIDAdminUserAuthenticated: {
+		Name:               "Admin API user authenticated",
+		Description:        "Admin API user successfully authenticated",
+		MandatoryFields:    AuditFields{},
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDAdminUserAuthenticationFailed: {
+		Name:        "Admin API user authentication failed",
+		Description: "Admin API user failed to authenticate",
+		MandatoryFields: AuditFields{
+			"username": "username",
+		},
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDAdminUserAuthorizationFailed: {
+		Name:        "Admin API user authorization failed",
+		Description: "Admin API user failed to authorize",
+		MandatoryFields: AuditFields{
+			"username": "username",
+		},
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDSyncGatewayCollectInfoStatus: {
+		Name:               "sgcollect_info status",
+		Description:        "sgcollect_info status was viewed",
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		MandatoryFields:    AuditFields{},
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDSyncGatewayCollectInfoStart: {
+		Name:               "sgcollect_info start",
+		Description:        "sgcollect_info was started",
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		MandatoryFields: AuditFields{
+			"output_dir":   "output_directory",
+			"upload_host":  "upload_host",
+			"customer":     "customer",
+			"ticket":       "ticket",
+			"keep_zip":     true,
+			"zip_filename": "zip_filename",
+		},
+		EventType: eventTypeAdmin,
+	},
+	AuditIDSyncGatewayCollectInfoStop: {
+		Name:               "sgcollect_info stop",
+		Description:        "sgcollect_info was stopped",
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		MandatoryFields:    AuditFields{},
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDSyncGatewayStats: {
+		Name:               "stats requested",
+		Description:        "stats were requested",
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDSyncGatewayProfiling: {
+		Name:        "profiling requested",
+		Description: "profiling was requested",
+		MandatoryFields: AuditFields{
+			"profile_type": "cpu, memory, etc.",
+		},
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDClusterInfoRead: {
+		Name:               "Sync Gateway cluster info read",
+		Description:        "Sync Gateway cluster info was viewed",
+		EnabledByDefault:   true,
+		FilteringPermitted: false,
+		EventType:          eventTypeAdmin,
+	},
+	AuditIDCreateDatabase: {
+		Name:        "Create database",
+		Description: "A new database was created",
+		MandatoryFields: AuditFields{
+			"db":     "database name",
+			"config": "JSON representation of db config",
+		},
```

Comment:
Should we log whether this is persistent config or not?

Reply:
Don't think it matters. Either way a database was created and the config of that database is available to see.